src/components/UpgradeTableOfContents.tsx

```diff
@@ -1,47 +1,10 @@
 import React from "react"
-import { motion } from "framer-motion"
-import { Link } from "gatsby"
-import styled from "@emotion/styled"
+import { Box, Link, ListItem, UnorderedList } from "@chakra-ui/react"
 
 import type { Item as ItemTableOfContents } from "./TableOfContents"
 
 const customIdRegEx = /^.+(\s*\{#([A-Za-z0-9\-_]+?)\}\s*)$/
 
-const Aside = styled.aside`
-  padding: 0rem;
-  text-align: right;
-  margin-bottom: 2rem;
-  overflow-y: auto;
-`
-
-const OuterList = styled(motion.ul)`
-  list-style-type: none;
-  list-style-image: none;
```

Comment:
I have no idea what this was supposed to do, but it also appeared to be doing nothing when I tested removing it—so I just left it out as I wasn't sure the best way to approach this in Chakra.